build/NuGetAdditionalFiles/RunCsc.cmd

```diff
@@ -4,4 +4,4 @@ if defined DOTNET_HOST_PATH (
 ) else (
     set HOST_PATH=dotnet
 )
-%HOST_PATH% %~dp0\csc.dll %*
+"%HOST_PATH%" "%~dp0\csc.dll" %*
```

Comment:
What if `%HOST_PATH%` already has `"` surrounding it? The double double quoting will cause issues correct?

Reply:
I will try and see

Reply:
It seems like the double quotes are stripped to one set and produce the correct result.